README.md

```diff
@@ -8,6 +8,39 @@ PostRPC is distributed as two separate javascript bundles. One for the server, a
 
 ![Example](/example.png?raw=true "Example with Logging")
 
+## Usage
+
+[Server](#postrpcserver)
+
+[Client](#postrpcclient)
+
+[Generator](#generating-stubs-from-interface-definitions)
+
+## Development
+
+This project is a [Lerna](https://github.com/lerna/lerna) Monorepo, so consult the Lerna docs for command-specific information.
+
+Server and Client libraries are built by webpack as UMD modules.
+
+To develop locally:
+
+```sh
+# prepare project
+git clone ... && cd post-rpc
+lerna bootstrap --hoist
+
+# build project
+lerna run build
+lerna run build-dev # (needs work though)
```

Comment:
why do you need to run both?

Reply:
you don't, but maybe one day you will...?

Reply:
ok, I see how that is confusing. I'll remove it.